backend/index.js

```diff
@@ -76,7 +76,7 @@ app.get('/health', (req, res, next) => {
 
 app.get('/ping', (req, res) => res.send('pong'))
 
-app.get('/info', (req, res) => res.send({ productId: config.productId }))
+app.get('/info', (req, res) => res.send({ productId: config.serviceName }))
```

Comment:
`config.serviceName` doesn't exist. Everywhere else it's been referenced as `config.apis.audit.serviceName`. Is it  better to define it at the top level and get the `config.apis.audit.serviceName` just use that? The `serviceName` is a constant though and would never change so could it just be in constants instead and referenced everywhere from there? The environment variable AUDIT_SERVICE_NAME would override the audit service name if we wanted it to be different from the actual service name, but this could be defaulted to the constant too